charts/yurt-manager/crds/apps.openyurt.io_nodepools.yaml

```diff
@@ -290,6 +290,195 @@ spec:
+                        type: string
+                      timeAdded:
+                        description: |-
+                          TimeAdded represents the time at which the taint was added.
+                          It is only written for NoExecute taints.
+                        format: date-time
+                        type: string
+                      value:
+                        description: The taint value corresponding to the taint key.
+                        type: string
+                    required:
+                      - effect
+                      - key
+                    type: object
+                  type: array
+                type:
+                  description: The type of the NodePool
+                  type: string
+              type: object
+            status:
+              description: NodePoolStatus defines the observed state of NodePool
+              properties:
+                conditions:
+                  description: |-
+                    Conditions represents the latest available observations of a NodePool's
+                    current state that includes LeaderHubElection status.
+                  items:
+                    properties:
+                      lastTransitionTime:
+                        description: Last time the condition transitioned from one status to another.
+                        format: date-time
+                        type: string
+                      message:
+                        description: A human readable message indicating details about the transition.
+                        type: string
+                      reason:
+                        description: The reason for the condition's last transition.
+                        type: string
+                      status:
+                        description: Status of the condition, one of True, False, Unknown.
+                        type: string
+                      type:
+                        description: Type of NodePool condition.
+                        type: string
+                    type: object
+                  type: array
+                leaderEndpoints:
+                  description: LeaderEndpoints is used for storing the address of Leader Yurthub.
+                  items:
+                    type: string
+                  type: array
+                nodes:
+                  description: The list of nodes' names in the pool
+                  items:
+                    type: string
+                  type: array
+                readyNodeNum:
+                  description: Total number of ready nodes in the pool.
+                  format: int32
+                  type: integer
+                unreadyNodeNum:
+                  description: Total number of unready nodes in the pool.
+                  format: int32
+                  type: integer
+              type: object
+          type: object
+      served: true
+      storage: true
```

Comment:
Yes, I will configure `v1beta2` as storage for NodePool, so we need to modify `v1beta1` storage to `false.`

https://github.com/openyurtio/openyurt/blob/9ba1d5e6d94e4f25687468855922c18527479154/pkg/apis/apps/v1beta1/nodepool_types.go#L82